python/cross_service/textract_comprehend_notebook/test-texttract-comprehend.py

```diff
@@ -0,0 +1,41 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+from testbook import testbook
+
+@testbook('TextractAndComprehendNotebook.ipynb', execute=[6, 8])
+def test_text_detection(tb):
+    detect_func = tb.ref("process_text_detection")
+    # Replaces aws_access_code value with your AWS access code, the value of aws_secret_code with your secret code,
+    # and the value of region with the Region assigned to your IAM user role
+    aws_access_code = "your access code here"
+    aws_secret_code = "your secret access code here"
+    region = "your resource region here"
```

Comment:
This is OK, and it works, but it's still a little risky putting credentials in code because you might accidentally push them. You could instead use Boto3 to get default credentials out of the user's credentials file:

```
import boto3
session = boto3.Session()
creds = session.get_credentials()
res = detect_func(bucket, document, creds.access_key, creds.secret_key, session.region_name)
```